test/fixture.h

```diff
@@ -148,6 +151,7 @@ class GraftServerTestBase : public ::testing::Test
         std::atomic<graft::HttpConnectionManager*> phttpcm{nullptr};
     private:
         std::thread th;
+        graft::supernode::system_info::Counter sys_info_;
```

Comment:
can we avoid pushing this changes until we agree on them?

Reply:
We can come to better solution later. But now it give us working tests.